src/extras/MovieClip.js

```diff
@@ -170,8 +203,16 @@ MovieClip.prototype.gotoAndStop = function (frameNumber)
 
     this._currentTime = frameNumber;
 
-    var round = Math.floor(this._currentTime);
-    this._texture = this._textures[round % this._textures.length];
+    if (this._durations !== null)
+    {
+        this._lag = frameNumber % 1 * this._durations[this.currentFrame];
```

Comment:
Doesn't this always equal `0`? Anything mod 1 is 0, and since mod and multiplication are in the [same operator precedence](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence) this just runs left-to-right making it `0 * value`.